examples/analytics-error-reporting.amp.html

```diff
@@ -48,36 +51,58 @@
     Duis et nisi sed urna blandit bibendum et sit amet erat.
 </p>
 
+<h2>amp-pixel fake 1p error</h2>
 <amp-pixel src="https://foo.com/tracker/foo"
            layout="nodisplay"
            referrerpolicy="referrer-fail"></amp-pixel>
 
-<h2 id="amp-iframe">IFrame</h2>
+<h2 id="amp-iframe">IFrame fake 1p error</h2>
 <amp-iframe width=10 height=10
             layout="responsive"
             src="https://www.google.com/maps/embed/v1/place?key=AIzaSyDG9YXIhKBhqclZizcSzJ0ROiE0qgVfwzI&q=Alameda,%20CA">
 </amp-iframe>
 
+<h2>amp-list fake 1p error</h2>
 <amp-list layout="responsive" src= [src]="src" width="600" height="600">
     <template type="amp-mustache">
     </template>
 </amp-list>
 
+<h2>amp-youtube fake 1p error</h2>
 <amp-youtube width="480" height="270"
              data-param-autoplay
              data-videoid="dQw4w9WgXcQ">
 </amp-youtube>
 
-<amp-analytics type="googleanalytics">
+<h1>These create 3P error</h1>
+<h2>A9 fake 3p error</h2>
```

Comment:
I would prefer throw fake error in our fake _ping_ ad. But I am ok with using these two, if they return use error.

Reply:
yeah these return user error. I can add more errors if needed?